taxonomies/labels.txt

```diff
@@ -242,14 +263,14 @@ en:Free range chicken
 de:Freilandhühnchen
 fi:ulkokana, vapaana elänyt kana
 hu:Szabad tartású csirke, Szabad tartású tyúk
-pt:Galinhas criadas em liberdade
+pt:Galinhas criadas em liberdade, galinhas em liberdade, galinhas livres de gaiolas, galinhas criadas ao ar livre, galinhas ao ar livre
```

Comment:
galinhas ao ar livre can be a  synonym for en:free range chicken or en:free range poultry, but not both.